### PR TITLE
docs/user-guide: document the new installer.modules customization

### PR DESCRIPTION
**Pull request by achilleas-k**
Introduced in https://github.com/osbuild/images/pull/799
Added to bootc-image-builder in https://github.com/osbuild/bootc-image-builder/pull/590

### CODE REVIEW

File: docs/user-guide/01-blueprint-reference.md [hunk 1]
```diff
@@ -1399,6 +1399,57 @@ network --bootproto=dhcp --device=link --activate --onboot=on
...
+
+The Anaconda installer can be configured by enabling or disabling its dbus modules.
+
+<Tabs values={tabValues} >
+<TabItem value="on-premises" >
+```toml
+[customizations.installer.modules]
+enable = []
+disable = []
+"""
```
Comment (thozza) [inline docs/user-guide/01-blueprint-reference.md]: is this a typo?

Comment (achilleas-k) [reply docs/user-guide/01-blueprint-reference.md]: It was.  Thanks!

**Review by thozza — DISMISSED** [review]
LGTM

**Comment by thozza** [comment]
@achilleas-k could be marked as ready, no?

**Comment by achilleas-k** [comment]
The original version of this PR had the feature documented for osbuild-composer only.  Right now, it's only supported in bootc-image-builder.  I didn't manage to add it to osbuild-composer yet.

**Comment by achilleas-k** [comment]
Marking ready.  Will update the osbuild-composer part when it's added.

**Review by thozza — DISMISSED** [review]
LGTM

File: docs/user-guide/01-blueprint-reference.md [hunk 2]
```diff
@@ -1521,6 +1521,67 @@ network --bootproto=dhcp --device=link --activate --onboot=on
 Note that osbuild-composer will automatically add the command that installs the system (`liveimg` or `ostreesetup`) if one is relevant for the image type (`image-installer` and `edge-installer`/`iot-installer` respectively), so this line or any line that conflicts with it should not be included. See the relevant [Kickstart documentation](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer) for more information.
 This customization cannot be used in combination with any other installer customizations.
 
+#### Anaconda Modules 🟣 {#installer-kickstart}
+
+The Anaconda installer can be configured by enabling or disabling its dbus modules.
+
+<Tabs values={tabValues} >
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: Please change to `tabValuesBootcOnly` and make `bootc` the default
```suggestion
<Tabs values={tabValuesBootcOnly} defaultValue="bootc">
```

where you need to implement this in `src/components/Highlight.js` first and import here 😅
```
export const tabValuesBootcOnly = [
    { label: '⚪ on premises', value: 'on-premises', attributes: { className: 'highlight tab disabled' } },
    { label: '⚪ hosted', value: 'hosted', attributes: { className: 'highlight tab disabled' } },
    { label: '🟣 bootc', value: 'bootc', attributes: { className: 'highlight tab bootc' } },
];
```

File: docs/user-guide/01-blueprint-reference.md [hunk 3]
```diff
@@ -1521,6 +1521,67 @@ network --bootproto=dhcp --device=link --activate --onboot=on
 Note that osbuild-composer will automatically add the command that installs the system (`liveimg` or `ostreesetup`) if one is relevant for the image type (`image-installer` and `edge-installer`/`iot-installer` respectively), so this line or any line that conflicts with it should not be included. See the relevant [Kickstart documentation](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer) for more information.
 This customization cannot be used in combination with any other installer customizations.
 
+#### Anaconda Modules 🟣 {#installer-kickstart}
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: The anchor needs to be unique e.g.
```suggestion
#### Anaconda Modules 🟣 {#installer-kickstart-anaconda-modules}
```

File: docs/user-guide/01-blueprint-reference.md [hunk 4]
```diff
@@ -1521,6 +1521,67 @@ network --bootproto=dhcp --device=link --activate --onboot=on
...
+
+The Anaconda installer can be configured by enabling or disabling its dbus modules.
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: nitpick - is it DBus or D-Bus?
```suggestion
The Anaconda installer can be configured by enabling or disabling its D-Bus modules.
```

File: docs/user-guide/01-blueprint-reference.md [hunk 5]
```diff
@@ -1521,6 +1521,67 @@ network --bootproto=dhcp --device=link --activate --onboot=on
...
+- `org.fedoraproject.Anaconda.Modules.Payloads`
+- `org.fedoraproject.Anaconda.Modules.Runtime`
+- `org.fedoraproject.Anaconda.Modules.Security`
+- `org.fedoraproject.Anaconda.Modules.Services`
+- `org.fedoraproject.Anaconda.Modules.Storage`
+- `org.fedoraproject.Anaconda.Modules.Subscription`
+- `org.fedoraproject.Anaconda.Modules.Timezone`
+- `org.fedoraproject.Anaconda.Modules.Users`
+
+*Note: The values are not validated. Any name listed under `enable` will be added to the Anaconda configuration. This way, new or unknown modules can be enabled. However, it also means that mistyped or incorrect values may cause Anaconda to fail to start.*
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: maybe emphasize this more?
```suggestion
*Note: The values are **not validated**. Any name listed under `enable` will be added to the Anaconda configuration. This way, new or unknown modules can be enabled. However, it also means that mistyped or incorrect values may cause Anaconda to fail to start.*
```

**Review by schuellerf — CHANGES_REQUESTED** [review]

File: docs/user-guide/01-blueprint-reference.md (hunk 2 above)
Comment (achilleas-k) [reply docs/user-guide/01-blueprint-reference.md]: Very nice, thanks.  Added!

File: docs/user-guide/01-blueprint-reference.md (hunk 5 above)
Comment (achilleas-k) [reply docs/user-guide/01-blueprint-reference.md]: Good idea.

**Review by schuellerf — APPROVED** [review]
Thanks for the changes, lgtm

**Review by thozza — APPROVED** [review]
I knew that Florian will have some useful comments :innocent: LGTM!

**Pull request merged**